inst/www/vertical-tab-panel/vertical-tab-panel.js

```diff
@@ -4,7 +4,7 @@
 /* Source: https://bootsnipp.com/snippets/featured/simple-vertical-tab */
 
 $(document).ready(function() {
-    $("div.bhoechie-tab-menu>div.list-group>a").click(function(e) {
```

Comment:
Note: I believe that without this edit, verticalTabsetPanel will not work if inserted via insertUI.